flink-formats/flink-avro-glue-schema-registry/src/main/java/org/apache/flink/formats/avro/glue/schema/registry/GlueSchemaRegistryAvroSchemaCoderProvider.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.formats.avro.glue.schema.registry;
+
+import org.apache.flink.formats.avro.SchemaCoder;
+
+import lombok.NonNull;
```

Comment:
I think wrong annotation

Reply:
The `NonNull` annotation is to make sure that users don't pass null for `configs`

Reply:
Yes I meant you are using `lombok` rather than `javax.annotation`. But I think this is not needed since Flink coding standards say everything is `nonnull` by default:
- https://flink.apache.org/contributing/code-style-and-quality-common.html#nullability-of-the-mutable-parts

This also opens the question as to why the `transportName` is not `@NonNull`? Is it `@Nullable`? 

Please remove the annotations unless there is a good reason to keep them.

Reply:
`transportName` can be null because we've set default value for it.

Reply:
Ah ok, Flink convention is that `@Nonnul` is default and `@Nullable` fields should be annotated. Please change accordingly